tox.ini

```diff
@@ -37,25 +30,35 @@ deps =
     readme
 
 [testenv:py27-pytest]
-commands = pytest {[tox]test_dir}
+deps =
+    -r requirements.txt
+    pytest
+    docker
+commands = pytest -v {[tox]test_dir}
 
 [testenv:py35-pytest]
-passenv = TRAVIS,CODACY_PROJECT_TOKEN
+deps =
+    -r requirements.txt
+    pytest
+    pytest-cov
+    coverage
+    codacy-coverage
+    docker
 whitelist_externals = sed
                       bash
 commands =
-    pytest --cov={envsitepackagesdir}/ephemeris --cov-report xml {[tox]test_dir}
+    pytest -v --cov={envsitepackagesdir}/ephemeris --cov-report xml {[tox]test_dir}
```

Comment:
wouldn't this work ?
```
pytest -v --cov src --cov-report xml
```

it does work locally for me, and I think you won't have to `sed` the xml file

Reply:
it seems to:

```
$ tox -e py36-pytest
GLOB sdist-make: /Users/mvandenb/src/ephemeris/setup.py
py36-pytest create: /Users/mvandenb/src/ephemeris/.tox/py36-pytest
py36-pytest installdeps: -rrequirements.txt, pytest, pytest-cov, coverage, codacy-coverage, docker
py36-pytest inst: /Users/mvandenb/src/ephemeris/.tox/dist/ephemeris-0.9.1.dev0.zip
py36-pytest installed: atomicwrites==1.1.5,attrs==18.1.0,bioblend==0.11.0,boto==2.49.0,certifi==2018.8.13,chardet==3.0.4,codacy-coverage==1.3.11,coverage==4.5.1,docker==3.5.0,docker-pycreds==0.3.0,docutils==0.14,ephemeris==0.9.1.dev0,galaxy-lib==18.5.13,idna==2.7,Jinja2==2.10,MarkupSafe==1.0,more-itertools==4.3.0,packaging==17.1,pluggy==0.7.1,py==1.5.4,pyparsing==2.2.0,pytest==3.7.2,pytest-cov==2.5.1,PyYAML==3.13,requests==2.19.1,requests-toolbelt==0.8.0,six==1.11.0,urllib3==1.23,websocket-client==0.51.0
py36-pytest runtests: PYTHONHASHSEED='1055593875'
py36-pytest runtests: commands[0] | pytest -v --cov=ephemeris --cov-report xml
========================================================================== test session starts ===========================================================================
platform darwin -- Python 3.6.5, pytest-3.7.2, py-1.5.4, pluggy-0.7.1 -- /Users/mvandenb/src/ephemeris/.tox/py36-pytest/bin/python3.6
cachedir: .pytest_cache
rootdir: /Users/mvandenb/src/ephemeris, inifile:
plugins: cov-2.5.1
collected 7 items

tests/test_run_data_managers.py::TestRunDataManagers::test_install_data_managers PASSED                                                                            [ 14%]
tests/test_run_data_managers.py::TestRunDataManagers::test_run_data_managers
```

Reply:
I take that back:
```
ERROR: Failed to generate report: No data to report.


ERROR: InvocationError for command '/Users/mvandenb/src/ephemeris/.tox/py36-pytest/bin/pytest -v --cov=ephemeris --cov-report xml' (exited with code 4)
```